coral/src/app/features/topics/details/documentation/components/documentation-view.module.css

```diff
@@ -0,0 +1,59 @@
+/* The html we show in ReactMarkdown does not contain */
+/* classes or style options. So elements are using the  */
+/* global styles that are based on the DS styles. Since these */
+/* are build in a way where styles should not rely on elements,  */
+/* e.g. h1 has the same font-size as body.  */
+/* We need to overwrite styles for elements in the markdown */
+/* context to show user a more refined documentation based */
+/* on their input.  */
+
+/*TYPOGRAPHY*/
+
+.markdownEditor h1,
+.markdownEditor h2,
+.markdownEditor h3 {
+  font-family: "Inter", serif;
+  font-weight: 700;
+  color: var(--headline-text-color);
+  /* setting margins since user can't style otherwise
+  and it should fit with the spacing for paragraphs */
+  margin-top: 0.5rem;
+  margin-bottom: 0.5rem;
```

Comment:
We can't use the aquarium variables here directly unfortunately. Same as `--headline-text-color` we would either have to overwrite the existing variables (*) or create new variables, which would not really give as any benefit. Also, aquarium does not really have the concept of "this is a paragraph so it should have spacings like this", so it would not even be a "real" relation between style and element.  

Headline and body colour size do have a direct relation, that's why I used the variables there (although they are also not aquarium variables). text body colour was already needed for another style, so I thought doing the same with headline text colour would be more consistent.

(*)  and that would mean we would potentially derive from DS in styles by mistake if a variable there changes without us noticing.